src/notification/notification.component.ts

```diff
@@ -124,11 +120,12 @@ export class Notification {
 		this.close.emit();
 	}
 
-	/**
-	 * Emits action event.
-	 */
-	onAction() {
-		this.action.emit();
+	onClick(action) {
+		if (typeof action === "function") {
```

Comment:
`isObservable` may be more reliable ... Also we'll want to check if the callback/observable even exists first and bail out if it doesn't (not sure why someone would do that, but people do strange things)

Reply:
isObservable will always return false because after subscribing to the action subject it is no longer an observable and becomes of type object instead.

Reply:
Hmm I don't think that should happen 🤔 interesting